falcon/vendor/mimeparse/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+This module wraps code from the MIT-licensed python-mimeparse project. The
+original project source code may be obtained from GitHub by visiting the
+following URL:
+
+    https://github.com/dbtsai/python-mimeparse
+
+"""
+
+from .mimeparse import *  # NOQA
```

Comment:
No new line at the end of file (in the Unix world, it is a punishable offense :smiling_imp: )

Reply:
Sheesh. Usually I add an extra line without thinking. Fixed.

Reply:
Editors normally add that, at least for source files. And `flake8` would complain, but `NOQA` on the last line felled us here. At least Github rendered it in a very visual way, making my job easy :slightly_smiling_face: